ebpf related warning during compilation

<!--
When creating a bug report please:
- Verify first that your issue is not already reported on GitHub.
- Test if the latest release and master branch are affected too.
-->

##### Bug report summary
<!-- Provide a clear and concise description of the bug you're experiencing. -->

I get the following warning during compilation, i do

> [ilyam@ilyam-pc ~]$ sudo ./netdata-installer.sh --install /opt --disable-cloud

```cmd
In function 'fill_resolved_name',
    inlined from 'fill_names.part.0' at collectors/ebpf.plugin/ebpf_socket.c:656:5:
collectors/ebpf.plugin/ebpf_socket.c:546:9: warning: 'memcpy' writing 1018 bytes into a region of size 980 [-Wstringop-overflow=]
  546 |         memcpy(ptr->resolved_name, hostname, length);
      |         ^
collectors/ebpf.plugin/ebpf_socket.c: In function 'fill_names.part.0':
libnetdata/libnetdata.c:177:15: note: at offset 0 to an object with size 980 allocated by 'malloc' here
  177 |     void *p = malloc(size);
      |               ^
In function 'fill_resolved_name',
    inlined from 'fill_names.part.0' at collectors/ebpf.plugin/ebpf_socket.c:656:5:
collectors/ebpf.plugin/ebpf_socket.c:547:36: warning: writing 1 byte into a region of size 0 [-Wstringop-overflow=]
  547 |         ptr->resolved_name[length] = '\0';
      |                                    ^
collectors/ebpf.plugin/ebpf_socket.c: In function 'fill_names.part.0':
libnetdata/libnetdata.c:177:15: note: at offset 1018 to an object with size 0 allocated by 'malloc' here
  177 |     void *p = malloc(size);
      |               ^
```
##### OS / Environment
<!--
Provide as much information about your environment (which operating system and distribution you're using, if Netdata is running in a container, etc.)
as possible to allow us reproduce this bug faster.

To get this information, execute the following commands based on your operating system:
- uname -a; grep -Hv "^#" /etc/*release  # Linux
- uname -a; uname -K                     # BSD
- uname -a; sw_vers                      # macOS

Place the output from the command in the code section below.  
 -->
```
[ilyam@ilyam-pc ~]$ uname -a; grep -Hv "^#" /etc/*release
Linux ilyam-pc 5.7.9-1-MANJARO #1 SMP PREEMPT Thu Jul 16 08:20:05 UTC 2020 x86_64 GNU/Linux
/etc/arch-release:Manjaro Linux
/etc/lsb-release:DISTRIB_ID=ManjaroLinux
/etc/lsb-release:DISTRIB_RELEASE=20.0.3
/etc/lsb-release:DISTRIB_CODENAME=Lysia
/etc/lsb-release:DISTRIB_DESCRIPTION="Manjaro Linux"
/etc/manjaro-release:Manjaro Linux
/etc/os-release:NAME="Manjaro Linux"
/etc/os-release:ID=manjaro
/etc/os-release:ID_LIKE=arch
/etc/os-release:BUILD_ID=rolling
/etc/os-release:PRETTY_NAME="Manjaro Linux"
/etc/os-release:ANSI_COLOR="32;2;24;144;200"
/etc/os-release:HOME_URL="https://manjaro.org/"
/etc/os-release:DOCUMENTATION_URL="https://wiki.manjaro.org/"
/etc/os-release:SUPPORT_URL="https://manjaro.org/"
/etc/os-release:BUG_REPORT_URL="https://bugs.manjaro.org/"
/etc/os-release:LOGO=manjarolinux
```

##### Netdata version
<!--
Provide output of `netdata -V`.
 
If Netdata is running, execute: $(ps aux | grep -E -o "[a-zA-Z/]+netdata ") -V
 -->
 
```cmd
[ilyam@ilyam-pc ~]$ /opt/netdata/usr/sbin/netdata -v
netdata v1.23.2-114-ge8403221
```

##### Component Name
<!--
Let us know which component is affected by the bug. Our code is structured according to its component,
so the component name is the same as the top level directory of the repository.
For example, a bug in the dashboard would be under the web component.
-->

ebpf

##### Steps To Reproduce
<!--
Describe how you found this bug and how we can reproduce it, preferably with a minimal test-case scenario.
If you'd like to attach larger files, use gist.github.com and paste in links.
-->

1. install via `netdata-installer.sh` script
2. ...

##### Expected behavior
<!-- Provide a clear and concise description of what you expected to happen. -->

No warning during compilation

--- a/collectors/ebpf.plugin/ebpf_socket.c
+++ b/collectors/ebpf.plugin/ebpf_socket.c
@@ -542,7 +542,7 @@ static inline void fill_resolved_name(netdata_socket_plot_t *ptr, char *hostname
         ptr->resolved_name = strdupz(hostname);
     else {
         length = NETDATA_MAX_NETWORK_COMBINED_LENGTH;
-        ptr->resolved_name = mallocz(NETDATA_DIM_LENGTH_WITHOUT_SERVICE_PROTOCOL + 1);
+        ptr->resolved_name = mallocz( NETDATA_MAX_NETWORK_COMBINED_LENGTH + 1);
         memcpy(ptr->resolved_name, hostname, length);
         ptr->resolved_name[length] = '\0';
     }

